agent_payload.proto

```diff
@@ -59,6 +61,28 @@ message EventsPayload {
 	CommonMetadata metadata = 2;
 }
 
+message SketchPayload {
+        message Summary {
+                message Sketch {
+                        message Entry {
+                                double v = 1;
+                                int64 g = 2;
+                                int64 delta = 3;
+                        }
```

Comment:
Splitting out the entry fields would allow [packed encoding](https://developers.google.com/protocol-buffers/docs/encoding#packed). So serialization should be much much faster (and make the payload much smaller).

e.g.
```
message Quantile {
  repeated double v = 1;
  repeated int64 g = 2;
  repeated int64 delta = 3;
  
  double min = 4;  
  double max = 5;
  double sum = 6;
  double avg = 7;
  int64 count = 8;
}```